tests/subsys/host_command/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  host_command.core:
+    platform_whitelist: native_posix
```

Comment:
platform_allow :)

Reply:
Thank you! You can tell I started this CL a little while ago, before that change. I updated locally, but will wait to push another set of patches until we can decide on directory naming